R/test_ranges.R

```diff
@@ -68,7 +70,7 @@ test_ranges <- function(data, tests, reaction = c('message', 'stop'),
     low_data <- if (!is.null(low) && any(data_variables < low)) {
       data_variables %>%
         magclass_to_tibble() %>%
-        filter(.data$value < low) %>%
+        filter(.data$value < low - tolerance) %>%
```

Comment:
If my understanding is correct, the tolerance should be added instead of subtracted, no? In an example with the lower threshold being 0 we would want to add a tolerance of the lower threshold being positive 1^-10 instead I think?

Reply:
Makes sense, I got it the wrong way around!